elasticsearch/kibana/install.sls

```diff
@@ -1,7 +1,12 @@
 {% from "elasticsearch/kibana/map.jinja" import kibana with context %}
 
+include:
+  - elasticsearch.repository
+
 install_kibana:
   pkg.installed:
     - pkgs: {{ kibana.pkgs }}
     - reload_modules: True
-    - update: True
\ No newline at end of file
+    - update: True
+    - require:
+      - pkgrepo: configure_elasticsearch_package_repo
```

Comment:
Oh, the comment is a question and not a statement. I sense progress in my skills :-)
Based on my reading of it, I don't think so since the logic is placed in the configure_elasticsearch_package_repo for adding the appropriate repo based on whether elastic_stack is True or False